pkg/sql/sem/builtins/builtins.go

```diff
@@ -111,6 +112,7 @@ const (
 	categoryArray               = "Array"
 	categoryComparison          = "Comparison"
 	categoryCompatibility       = "Compatibility"
+	categoryCrypto              = "Cryptographic"
```

Comment:
wonder if we should move some functions to this category. not important for this PR though.